Firestore/Protos/CMakeLists.txt

```diff
@@ -111,9 +111,6 @@ endfunction()
 #
 # Libprotobuf Objective-C also includes the well-known protos so they must be
 # omitted here.
-#
-# The Objective-C client also uses the generated gRPC stubs for the Firestore
-# service.
```

Comment:
If we've removed the objc grpc stubs, don't we now need the cpp ones?

Reply:
If we can do this without generated code that's better. The generated code is essentially dead-weight given the way we're doing our own serialization so let's avoid it.